.maintain/gitlab/check_polkadot_companion_build.sh

```diff
@@ -78,12 +82,14 @@ then
     boldprint "companion pr specified/detected: #${pr_companion}"
     git fetch --depth 1 origin refs/pull/${pr_companion}/head:pr/${pr_companion}
     git checkout pr/${pr_companion}
+    git merge origin/master
```

Comment:
when fetching with `--depth 1` the master branch won't be available in this repo. this will require something like [this](https://github.com/paritytech/substrate/blob/master/.maintain/gitlab/check_runtime.sh#L34) before.

Reply:
(this is the case for all merges here, since `GIT_DEPTH` is set to only do shallow cloning)